CONTRIBUTING.md

```diff
@@ -30,6 +30,26 @@ We use GitHub templates to manage the requests. You will either select:
 
 Ensure the appropriate label is selected before submitting your issue.
 
+## Reviewing Pull Requests
+
+Each pull request requires at least *one* review and approval from an active
+contributor. The primary function of reviews is to ensure code contributors
+are not blocked.
+
+We ask that reviewers perform a basic sanity check on all code and documentation
+contributions. 
+
+All pull requests should be reviewed, even if the request is
+tagged with the label `needs-rebase`. At this time, it is imperative that
+code contributions move swiftly, and without minor blockers.
+
+Note: If the pull request requires a rebase, it is suitable to approve the
+pull request, the pull request will not merge automatically.
+
+The only pull requests that do not require a review is anything in draft status,
```

Comment:
I'd argue that even draft PRs should be reviewed if a review is requested.

IMO, if a PR is opened is because the author wants/needs feedback, even if it's a draft. Same goes for WIP.

Reply:
The keyword is "require"; draft PRs don't need one (but can request feedback). But then, draft/WIP won't get merged, and will require a mandatory review prior to merge, hence this may be redundant. Maybe rephrase paragraph to:

```suggestion
If your pull request is not yet in the state where it can be meaningfully reviewed and merged, please leave it in draft status or tag with the `WIP` label. You can of course still ask for feedback by manually assigning a reviewer or pinging them directly at this stage.
```